airbyte-integrations/connectors/source-gong/source_gong/manifest.yaml

```
@@ -80,6 +80,70 @@ definitions:
         type: InlineSchemaLoader
         schema:
           $ref: "#/schemas/calls"
+    extensiveCalls:
+      type: DeclarativeStream
+      name: "extensiveCalls"
+      primary_key: [["id"]]
```

Should this be like this? 
```suggestion
      primary_key:
        - id
```